doc/rest_api/snippet_sharing/api-description.apib

```diff
@@ -0,0 +1,587 @@
+FORMAT: 1.0
+
+# REST service API for sharing configuration snippets
+
+This is a description of the API provided by the REST service that can be used to share, search and download configuration snippets.
+
+
+# API Description [/]
+
+## List API methods [GET]
+
+Returns an overview of available API methods with required parameters and a description.
+
++ Response 200 (text/html; charset=utf-8)
+
+
+## API Version [GET /version]
+
+Returns the currently used version of the API. A version consists of two numbers (format: a.b), where the first number (a) is the major version and the second number (b) the minor version. Minor versions may not contain breaking changes (i.e. backwards-compatibility is ensured), but major versions may. This means that minor versions may only extend the API.
```

Comment:
As stated below, this is not the Elektra version, but an own version number for the API. But your idea of using the Elektra version could work as well if same rules apply. The question is what to do if Elektra itself is not planning to bring out a bigger new version, but the API needs a breaking change?